orientdb/src/main/java/org/hibernate/datastore/ogm/orientdb/impl/OrientDBDatastoreProvider.java

```diff
@@ -46,58 +57,78 @@
 
 	@Override
 	public void start() {
-		LOG.info( "start" );
+		log.debug( "---start---" );
 		try {
 			PropertyReaderContext<String> jdbcUrlPropery = propertyReader.property( "javax.persistence.jdbc.url", String.class );
 			if ( jdbcUrlPropery != null ) {
-				String jdbcUrl = jdbcUrlPropery.getValue();
-				LOG.info( "jdbcUrl:" + jdbcUrl );
+				jdbcUrl = jdbcUrlPropery.getValue();
+				log.warn( "jdbcUrl:" + jdbcUrl );
 				Class.forName( propertyReader.property( "javax.persistence.jdbc.driver", String.class ).getValue() ).newInstance();
-				Properties info = new Properties();
+				info = new Properties();
 				info.put( "user", propertyReader.property( "javax.persistence.jdbc.user", String.class ).getValue() );
 				info.put( "password", propertyReader.property( "javax.persistence.jdbc.password", String.class ).getValue() );
```

Comment:
Generally speaking, dialects for NoSQL dbs don't use jdbc drivers. We have specific properties for OGM to read user and password. Check `OgmProperties.USERNAME` and `OgmProperties.PASSWORD`.